src/user/common.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { SinonSandbox } from 'sinon'
+import { Puppet, RoomPayload } from 'wechaty-puppet'
+
+export const mockRoomPayload = (
```

Comment:
I do not think coupling between unit tests is a good idea, and also it seems no strong reason for us to seperate this method to create new file.

Please move the code to the unit test file instead of creating this new file.

Reply:
The reason I do this is to make code climate happy, it was complaining about duplicate code in code base. I can revert the last commit if you think that's what you want.